### PR TITLE
Add ability to populate CATTLE_IP_SET

### PR DESCRIPTION
**Pull request by StrongMonkey**

### CODE REVIEW

**Comment by StrongMonkey** [comment]
https://github.com/rancher/rancher/issues/7034

**Comment by StrongMonkey** [comment]
https://github.com/rancher/rancher/issues/3703

File: agent/run.sh [hunk 1]
```diff
@@ -149,6 +149,7 @@ launch_agent()
 	 -e HTTPS_PROXY \
         -e NO_PROXY \
         -e no_proxy \
+	 -e CATTLE_IP_SET \
```
Comment (cjellick) [inline agent/run.sh]: CATTLE_SCHEDULER_IPS

**Comment by cjellick** [comment]
LGTM

**Pull request merged**